moped-editor/src/queries/components.js

```diff
@@ -30,65 +30,67 @@ export const ADD_PROJECT_COMPONENT = gql`
 `;
 
 export const GET_PROJECT_COMPONENTS = gql`
-  query GetProjectComponents($projectId: Int!) {
-    moped_proj_components(
-      where: { project_id: { _eq: $projectId }, is_deleted: { _eq: false } }
-    ) {
-      project_component_id
-      component_id
-      description
-      moped_components {
-        component_name
-        component_subtype
-        feature_layer {
-          internal_table
-        }
-        line_representation
-        moped_subcomponents {
-          subcomponent_id
-          subcomponent_name
-        }
+  fragment projectComponentFields on moped_proj_components {
```

Comment:
I pulled the common fields for project components out into a fragment so we can reuse them in the related project queries that are aliased as `parentProjectComponents`, `siblingProjects`, and `childProjects` below

Reply:
i like that a lot!